deepchem/pages/about.js

```diff
@@ -0,0 +1,102 @@
+import Image from "next/image";
+
+import DeveloperDeskImage from "../public/images/about/developer.png";
+import DeepForestLogo from "../public/images/about/companies-developing-deepchem/deepforest_logo.png";
+import StanfordLogo from "../public/images/about/companies-developing-deepchem/stanford-logo.png";
+
+const faqs = [
+  {
+    question: "What is DeepChem?",
```

Comment:
Can we move these FAQs into a text or markdown file and have the js load the text from the file? In general not good to maintain descriptive text directly in JS